pkg/networkservice/common/refresh/client_test.go

```diff
@@ -267,3 +268,24 @@ func TestRefreshClient_Sandbox(t *testing.T) {
 		return refreshSrv.getState() == testRefreshStateDoneRequest
 	}, sandboxTotalTimeout, sandboxStepDuration)
 }
+
+func TestRefreshClient_NoRefreshOnFailure(t *testing.T) {
```

Comment:
It seems to me this test is incorrect due to there are no expire time in request.
1. Please add expiration time and use `clockmock` to check that there are no new requests after getting an error on the request. 
2. Please add test for Close use-case. As I can see you've added a major return statement at pkg/networkservice/common/refresh/client.go:133. So this scenario should be covered.

Reply:
Yes, added.